### ISSUE TITLE
vercel_dns_record is not a valid resource type

### ISSUE THREAD

**Issue by barsteadr** [vercel/terraform-provider-vercel#36]
Based on the documentation, examples and code it looks like there should be support for the resource type `vercel_dns_record` yet using this resource type leads to this error:

```
The provider vercel/vercel does not support resource type "vercel_dns_record".
```
Is this a bug or will there soon be support for managing vercel DNS records?

**Comment by dglsparsons** [comment]
Hi @barsteadr. Thanks for raising this. Support for `vercel_dns_record` is merged into main but not yet released under a version. It'll be coming very soon! :)

**Comment by teleyinex** [comment]
Thanks for the follow up. I was having the same issue. Do you have an ETA?

**Comment by dglsparsons** [comment]
Out of curiosity, which documentation are you referring to? I don't see it as a resource [here](https://registry.terraform.io/providers/vercel/vercel/latest/docs). Was this looking at the source code directly or something else? 

@teleyinex Later today :)

**Comment by dglsparsons** [comment]
This is now released in [v0.5.0](https://github.com/vercel/terraform-provider-vercel/releases/tag/v0.5.0). 

Enjoy 🎊

**Comment by barsteadr** [comment]
@dglsparsons Yes I was just looking at the source code and didn't match up the release with the source. I was just starting this morning to manually copy over records from AWS Route 53 to Vercel. I'll now do this via terraform. Many thanks! 🎆
